pl_bolts/models/detection/components/torchvision_backbones.py

```diff
@@ -0,0 +1,102 @@
+
+import torch.nn as nn
+
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+try:
+    import torchvision
+
+    from ._supported_models import TORCHVISION_MODEL_ZOO
+except ModuleNotFoundError:
+    warn_missing_pkg('torchvision')  # pragma: no-cover
+
+
+__all__ = ["create_torchvision_backbone"]
+
+
+def _create_backbone_generic(model, out_channels: int):
+    """
+    Generic Backbone creater. It removes the last linear layer.
+    Args:
```

Comment:
Just realised that there should be an empty line right before `Args:` to properly render docs. #284
```suggestion
    Generic Backbone creater. It removes the last linear layer.

    Args:
```